regulations/static/regulations/css/less/layout.less

```diff
@@ -14,8 +14,8 @@ Main Content
 */
 
 .main-content {
-    margin-top: 50px;
-    padding: 50px 60px;
+    margin-top: @header_height;
+    padding: 40px 60px;
```

Comment:
Hmm, you're right. In my experimentation I didn't spend a lot of time adjusting the subhead height, but if you did, the top-padding would need to be adjusted in some cases.

I think the top padding should be even smaller and the top margin should include the subhead height. The padding on it's own doesn't really care about the headers, it just makes the page more readable. SO it makes sense to move more of the fixed-header-offsetting into the margin, so the padding just stays independent. Patch coming...